setup.sh

```diff
@@ -136,6 +138,8 @@ fi
 # you need to set up the Docker repository. Afterward, you can install and update Docker from the repository.
 # -----------------------------------------------------------------------------------------------------------
 
```

Comment:
Why is there no check for Docker at this point? You can skip all of the below if statements if Docker is already installed.

Reply:
## Note

Consideration for optimization down the road.